src/main/java/org/json/XMLParserConfiguration.java

```diff
@@ -54,6 +65,12 @@ public class XMLParserConfiguration {
      */
     private Set<String> forceList;
 
+    /**
+     * When parsing the XML into JSON, specifies the tags whose values should be converted
```

Comment:
This comment is incorrect (just a duplicate from the previous section) - suggested content:  
```
The maximum nesting depth when parsing a XML document to JSON.
```